content/about.md

```diff
@@ -1,49 +1,32 @@
----
+te---
 Title: "About"
 ---
-{{< project >}}
+OpenTelemetry provides the libraries, agents, and other components that you need to capture critical telemetry your production services so that you can better observe, manage, and debug them. Specifically, OpenTelemetry captures metrics, distributed traces, and resource metadata from your backend and client applications and then sends this data to backends like Prometheus, Jaeger, Zipkin, [and others](https://opentelemetry.io/registry/?s=exporter) for processing. OpenTelemetry is composed of the following:
+
+- One API and SDK per language, which include the interfaces and implementations that you need to define and create distributed traces and metrics, manage sampling and context propagation, etc.
+- Language-specific integrations for popular web frameworks, storage clients, RPC libraries, etc. that (when enabled) automatically capture relevant traces and metrics and handle context propagation
+- Language-specific exporters that allow SDKs to send captured traces and metrics to any supported backends
```

Comment:
nit: supported. We have core and contrib so supported may not be the right term. Not sure I have a better alternative.

Reply:
I can't think of a better alternative either. I'll leave this open for now.

Reply:
I think supported is fine for now. The distinction between core and contrib is probably not super important at this level of detail.